Assembler.Base/MessagesAssembler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.Enums;
+
+namespace Assembler.Base
+{
+    public class MessagesAssembler : IAssembler
+    {
+        private readonly IResolver<FrameType, IHandler<BaseFrame, BaseMessageInAssembly>> _resolver;
+        private readonly IConverter<BaseMessageInAssembly, BaseAssembledMessage> _converter;
```

Comment:
Consider more specific names for both fields above

Reply:
f

Reply:
👍